qcodes/instrument_drivers/tektronix/AWG5014.py

```diff
@@ -310,29 +337,6 @@ def __init__(self, name, setup_folder, address, reset=False,
                     vals=vals.Numbers(-2.7, 2.7),
                     get_parser=float)
 
-        # # Add functions
```

Comment:
What is the reason for removing the add_functions? As far as I understand this is a QCoDeS concept to easily list all the function (that are note parameters) in an instrument.